Kros.KORM/src/Kros.KORM/Materializer/DynamicMethodModelFactory.cs

```diff
@@ -39,21 +39,25 @@ public class DynamicMethodModelFactory : IModelFactory
         private readonly static MethodInfo _fnInjectorsListGetItem = typeof(List<IInjector>).GetProperty("Item").GetGetMethod();
         private readonly static MethodInfo _fnInjectorMethodInfo =
             typeof(IInjector).GetMethod(nameof(IInjector.GetValue), new Type[] { typeof(string) });
+        private readonly static Dictionary<string, MethodInfo> _readerValueGetters =
+            new Dictionary<string, MethodInfo>(StringComparer.CurrentCultureIgnoreCase);
+        private readonly static MethodInfo _getValueMethodInfo =
+            typeof(IDataRecord).GetMethod("GetValue", new Type[] { typeof(int) });
 
         #endregion
 
         #region Constructor
 
+        static DynamicMethodModelFactory() => InitReaderValueGetters();
```

Comment:
Static constructor is not needed. Static field can be initialized immediately by static method, which returns value:

```csharp
private readonly static Dictionary<string, MethodInfo> _readerValueGetters = InitReaderValueGetters();

private static Dictionary<string, MethodInfo> InitReaderValueGetters() {
    var getters = new Dictionary<string, MethodInfo>(StringComparer.CurrentCultureIgnoreCase);
    ...
    return getters;
}
```